NFT-RFC-002-use-cases.md

```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+
 
 #### User Needs
+User needs here are broken down for each of the three users presented in the above story: Ellenor, Vicki and Stephen. 
+
+__Outcome Funder__
+* Configuration of a clear set of criteria on which to base the outcome payments
+* Assurance that reported outcomes are properly validated by a clear and mutually agreed upon validation method
+* Mechanism to make the outcome payments as the various companies who are fronting the upfront costs reach desired outcome levels
+
+__Investing Companies__
+* Clear criteria for receiving outcome payments
+* Clear return on investment when the criteria for an outcome payment is met
+* Mechanism to receive the outcome payments once the company has reached the specified criteria
 
 #### Unique Challenges
+The metadata in this user story comes in when a company is reporting the outcome. This would be:
+* Where the serviced school is located
+* How many students will the upgraded labs service
+
+Given that the outcome funders want assurance that the company has properly completed the task for the outcome payment this means that the NFT generated from a token representing a set outcome claims will need to be:
+__Metadata Requirements:__ Mintable, Burnable, Ownable, Transferable, Lockable, Immutable, Composible
+
+In addition there is the interesting problem of how the outcome claims are verified. Likely this project would need buy-in from at least a set of school administrators who are willing to participate in the verification and feedback mechanisms that the outcome funders have decided they will need to have confidence in funding this project. 
 
 #### Distinctive Requirements
+Companies taking part in this project will want to be able to, in various circumstances, submit both outcome claims for a particular instance, i.e. one school has been serviced, while also having the ability to submit a group of claims at the same time. This will allow for the project to have flexible outcome payments which should enable companies both small, who may need to receive the ROI as soon as possible for their revenue stream, and large, who can whether the revenue hit until they finish a batch of projects, to both take part in the project. 
+
+#### NFT Metadata Layer Model
+| Layer |  |
+|-----|------|
+| **8. Operationalization** | Holder of the verified claim NFT submits a transaction to the bond chain and receive the associated payment |
+| **7. Assertions** | Verifiable claims are assertions by witnesses about the underlying impact claim, which constitute evidence for oracles. | 
+| **6. Extensions &    Restrictions to Rights** | N/A | 
```

Comment:
May be applicable based on the attributes of the beneficiary, for instance, age, gender, socio-economic status, ... which may be based on verifiable credentials